cirq/contrib/two_qubit_gates/gate_compilation_test.py

```diff
@@ -0,0 +1,70 @@
+"""Tests for gate_compilation.py"""
+import numpy as np
+import pytest
+
+from cirq import unitary, FSimGate, value
+from cirq.contrib.two_qubit_gates.gate_compilation import (
+    gate_product_tabulation)
+from cirq.contrib.two_qubit_gates.math_utils import (
+    unitary_entanglement_fidelity)
+from cirq.testing import random_special_unitary
+
+_rng = value.parse_random_state(11)  # for determinism
+
+sycamore_tabulation = gate_product_tabulation(unitary(
```

Comment:
How long does this test file take to run? It should be less than ten seconds, ideally less than a second.

Reply:
It takes about 0.6 seconds on my machine. I can lower the specs further if you like.

Reply:
0.6 seconds on my local machine. I can lower the specs further if you like.